docs/troubleshooting.md

```diff
@@ -1,8 +1,96 @@
 # Troubleshooting
 
-## dotnet is crashing
+## General steps
 
-Until version `v0.6.0-beta.1` (inclusive) there were issues instrumenting
+If you encounter any issue with OpenTelemetry .NET automatic instrumentation,
```

Comment:
it is against capitalization convention ; )

Reply:
fixed in https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/pull/2326/commits/5ab2e8fbdeced545c73045d37380285a09da95e0